test/tests.config.json

```diff
@@ -9,21 +9,26 @@
     "/scroll-animations/**/*.html"
   ],
   "excludeTests": [
-    "/scroll-animations/css/animation-shorthand.html",
-    "/scroll-animations/css/animation-timeline-computed.html",
-    "/scroll-animations/css/animation-timeline-in-keyframe.html",
-    "/scroll-animations/css/animation-timeline-none.html",
-    "/scroll-animations/css/animation-timeline-parsing.html",
-    "/scroll-animations/css/at-scroll-timeline-before-phase.html",
-    "/scroll-animations/css/at-scroll-timeline-cascade.html",
-    "/scroll-animations/css/at-scroll-timeline-element-offsets.html",
-    "/scroll-animations/css/at-scroll-timeline-inactive-phase.html",
-    "/scroll-animations/css/at-scroll-timeline-orientation.html",
-    "/scroll-animations/css/at-scroll-timeline-sampling.html",
-    "/scroll-animations/css/at-scroll-timeline-source.html",
-    "/scroll-animations/css/at-scroll-timeline-start-end.html",
-    "/scroll-animations/css/at-scroll-timeline-time-range.html",
-    "/scroll-animations/css/scroll-timeline-cssom.tentative.html"
+    "/scroll-animations/css/**/*.html",
```

Comment:
Please do not augment the test exclusions. We have other developers working on the Polyfill as well.  Fine to keep this as your own exclusions file, but this should not be part of the pull request.